e2e/process-services-cloud/start-task/start-task-group-cloud.e2e.ts

```diff
@@ -0,0 +1,194 @@
+        } catch (error) {
+        }
+        done();
+    });
+
+    beforeEach(() => {
+        loginSSOPage.loginSSOIdentityService(testUser.email, testUser.password);
+        navigationBarPage.navigateToProcessServicesCloudPage();
+        appListCloudComponent.checkApsContainer();
+        appListCloudComponent.checkAppIsDisplayed(simpleApp);
+        appListCloudComponent.goToApp(simpleApp);
+        tasksCloudDemoPage.taskListCloudComponent().getDataTable().waitForTableBody();
+        tasksCloudDemoPage.openNewTaskForm();
+        startTask.checkFormIsDisplayed();
+    });
+
+    afterEach(() => {
+        navigationBarPage.clickLogoutButton();
+    });
+
+    it('[C291954] Should be able to select/delete an group for a standalone task', () => {
+        peopleCloudComponent.clearAssignee();
+
+        groupCloud.searchGroups(testGroup.name);
+        groupCloud.checkGroupIsDisplayed(testGroup.name);
+        groupCloud.selectGroupFromList(testGroup.name);
+        groupCloud.checkSelectedGroup(testGroup.name);
+
+        groupCloud.searchGroups(hrGroup.name);
+        groupCloud.checkGroupIsDisplayed(hrGroup.name);
+        groupCloud.selectGroupFromList(hrGroup.name);
+        groupCloud.checkSelectedGroup(hrGroup.name);
+
+        groupCloud.removeSelectedGroup(testGroup.name);
+        groupCloud.checkNotSelectedGroup(testGroup.name);
+
+        startTask.addName(unassignedTaskName);
+        startTask.clickStartButton();
+
+        navigationBarPage.clickLogoutButton();
+        loginSSOPage.loginSSOIdentityService(apsUser.email, apsUser.password);
+        navigationBarPage.navigateToProcessServicesCloudPage();
+        appListCloudComponent.checkApsContainer();
+        appListCloudComponent.checkAppIsDisplayed(simpleApp);
+        appListCloudComponent.goToApp(simpleApp);
+        tasksCloudDemoPage.taskListCloudComponent().getDataTable().waitForTableBody();
+
+        tasksCloudDemoPage.editTaskFilterCloudComponent()
+            .clickCustomiseFilterHeader()
+            .clearAssignee()
+            .setStatusFilterDropDown('CREATED');
+
+        tasksCloudDemoPage.taskListCloudComponent().getDataTable().waitForTableBody();
+        tasksCloudDemoPage.taskListCloudComponent().checkContentIsDisplayedByName(unassignedTaskName);
+    });
+
+    it('[C291955] Should be able to select multiple groups when the selection mode=multiple', () => {
+        peopleCloudComponent.clearAssignee();
+
+        groupCloud.searchGroups(testGroup.name);
+        groupCloud.checkGroupIsDisplayed(testGroup.name);
+        groupCloud.selectGroupFromList(testGroup.name);
+        groupCloud.checkSelectedGroup(testGroup.name);
+
+        groupCloud.searchGroups(hrGroup.name);
+        groupCloud.checkGroupIsDisplayed(hrGroup.name);
+        groupCloud.selectGroupFromList(hrGroup.name);
+        groupCloud.checkSelectedGroup(hrGroup.name);
+
+        startTask.addName(unassignedTaskName);
+        startTask.clickStartButton();
```

Comment:
we also need to delete the default assigned user before we click on the start, to make sure, it's still accessible to the 2 group members.

Reply:
That's what we do line 140